test/.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+	"extends": "../src/.eslintrc.json",
+	"root": true,
```

Comment:
Yeah, you're right. :D How about we create `.eslintrc-(base|browser|node).json` files and extend whichever we need? Similarly to what I did for Angular; see https://github.com/angular/angular.js.